Formula/rgbds.rb

```diff
@@ -14,8 +14,11 @@ class Rgbds < Formula
     sha256 "c08b856ecf4cb57390ec99241d0bc87ba623536ab7e3e11d5cc23334230ff1cd" => :mavericks
   end
 
+  depends_on "pkg-config" => :build
```

Comment:
Think I figured out. It should work on every scenario now. Another time I will take a look at upstream to see if changes on the Makefile are possible concerning dependencies. Thanks again!